example/layer-examples.js

```diff
@@ -68,6 +68,8 @@ const ScatterplotLayerExample = props =>
   new ScatterplotLayer({
     data: props.points,
     opacity: 0.5,
+    outlineMode: true,
+    outlineStrokeWidth: 2,
```

Comment:
I believe that all the other layers that support line width prop just call it `strokeWidth`, so calling it that would be a little more consistent.